src/Type/BasePickerType.php

```diff
@@ -32,116 +31,205 @@
  */
 abstract class BasePickerType extends AbstractType implements LocaleAwareInterface
 {
+    /**
+     * @var array<string, array<string>|string>
+     */
+    private const DATEPICKER_ALLOWED_OPTIONS = [
+        'allowInputToggle' => 'bool',
+        'dateRange' => 'bool',
+        'debug' => 'bool',
+        'defaultDate' => ['null', 'string', \DateTimeInterface::class],
+        'keepInvalid' => 'bool',
+        'multipleDates' => 'bool',
+        'multipleDatesSeparator' => 'string',
+        'promptTimeOnDateChange' => 'bool',
+        'promptTimeOnDateChangeTransitionDelay' => 'integer',
+        'stepping' => 'integer',
+        'useCurrent' => 'bool',
+        'viewDate' => ['null', 'string', \DateTimeInterface::class],
+    ];
+
+    /**
+     * @var array<string, array<string>|string>
+     */
+    private const RESTRICTIONS_OPTIONS = [
+        'minDate' => ['null', 'string', \DateTimeInterface::class],
+        'maxDate' => ['null', 'string', \DateTimeInterface::class],
+        'disabledDates' => ['string[]', 'DateTimeInterface[]'],
+        'enabledDates' => ['string[]', 'DateTimeInterface[]'],
+        'daysOfWeekDisabled' => 'integer[]',
+        // 'disabledTimeIntervals' => [],
```

Comment:
I should check this property

Reply:
I will reomve this option for now, It does not seem to be working on tempus Dominus. Also I don't think it is to important.